backend/src/kitconcept/intranet/behaviors/configure.zcml

```diff
@@ -18,6 +18,20 @@
       provides=".blocks_settings.ITTWBlocksSettings"
       />
 
+  <plone:behavior
+      name="kitconcept.intranet.enable_likes"
+      title="Enable Like functionality"
+      description="Enables the like functionality for content objects"
+      provides=".enable_likes.IEnableLikes"
+      />
+
+  <plone:behavior
+      name="kitconcept.intranet.votes"
+      title="Saves userID who liked an item"
+      description="Adds votes field"
+      provides=".votes.IVotes"
+      />
```

Comment:
@iFlameing It seems a bit strange to me to have 2 different behaviors. It's not useful to enable one without enabling the other one, is it? Why not put both fields in the same behavior?

Reply:
I thought maybe we want some control over both. I have now put both of them in votes behaviors.